src/main/java/frc/robot/subsystems/claw/ClawStateMachine.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.claw;
+
+import frc.robot.lib.statemachine.StateMachine;
+
+/** Add your docs here. */
+public class ClawStateMachine extends StateMachine {
+    
+    public static OpenState openState = new OpenState();
```

Comment:
When we were discussing how the [claw state machine](https://lucid.app/lucidchart/b6090843-0b3b-4749-8b86-71403e67fac1/edit?invitationId=inv_7482c25f-2845-4fb0-9f3d-8a34f9c4208e&referringApp=slack&page=0_0#) works, we also included 'closing' and 'opening' states. These are not reflected in here.